packages/react/.storybook/theme.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { create } from '@storybook/theming';
+
+export default create({
+  base: 'light',
+  brandTitle: 'Carbon for IBM.com React',
+  brandUrl:
+    'https://github.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/react',
+  fontBase: '"IBM Plex Sans", "Helvetica Neue", Arial, sans-serif',
+  fontCode:
+    '"IBM Plex Mono", Menlo, "DejaVu Sans Mono", "Bitstream Vera Sans Mono", Courier, monospace',
+});
```

Comment:
Sorry, just noticed this after I made a comment in the issue! Thank you for opening this so quickly!

As mentioned, I'm wondering if we should have this centralized in the `storybook-addon-theme` package, so the theme can be shared between react and web components? I think changing other things like colors might be something worth looking into, and would be nice to manage that in one place.

Reply:
@jeffchew `storybook-addon-theme` is for the UI to choose Carbon color theme, and it doesn't have (or deal with) theme content at all. Sharing Storybook theme content is an interesting idea, though, we can try it once we get more Storybook theme content.

Reply:
Thanks, and I think this is probably a future conversation anyway, as discussed we will add this to the design team's ideation board for further research around this.